index.html

```diff
@@ -3,9 +3,25 @@
 <head>
   <meta charset="utf-8">
   <title>Quantified Self - Diary</title>
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
i suggest using webpack to load this in for you!

Reply:
I have webpack and my jquery doesn't work without this line of code